prov/gni/test/rdm_atomic.c

```diff
@@ -481,7 +484,7 @@ void do_min(int len)
 
 	cr_assert_eq(ret, 1, "fi_cq_read returned %d %d", ret,
 			dump_cq_error(send_cq[0], target, 0));
-	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0);
+	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0, ep[1]);
```

Comment:
In thinking about this a little more, I think we need to test both paths, with and without RMA events.  In reviewing the tests, if there is an obvious way to test both, let's do that.  It might just be a matter of factoring the set up function.